### ISSUE TITLE
Resources not found in jsdocs (HTTP 404)

### ISSUE THREAD

**Issue by cnsgithub** [primefaces/primefaces-showcase#231]
To reproduce:
1. Go to https://www.primefaces.org/showcase/ui/data/datatable/basic.xhtml
2. Click at 
![image](https://user-images.githubusercontent.com/13161711/101444398-b518c180-391f-11eb-854d-36b7f9ca513e.png)
3. Click at 
![image](https://user-images.githubusercontent.com/13161711/101444446-ce217280-391f-11eb-961b-8e183ec929a4.png)
4. For example, click at 
![image](https://user-images.githubusercontent.com/13161711/101444530-f3ae7c00-391f-11eb-887f-c151ff7c7352.png) or 
![image](https://user-images.githubusercontent.com/13161711/101444553-0032d480-3920-11eb-8fb4-9c07af7564b0.png)

You will get HTTP 404 errors, e.g. https://primefaces.github.io/primefaces/8_0/primefaces.widget.datatable.html#addrow
![image](https://user-images.githubusercontent.com/13161711/101444606-2193c080-3920-11eb-9aa6-9b0850fe4ce6.png)

**Comment by tandraschko** [comment]
@blutorange could you check this? Maybe related to my commit Yesterday? I runned the release Profile and updated the docs

**Comment by blutorange** [comment]
@tandraschko I'm currently fixing a few typing errors in the docs I noticed, will look at this as well.

**Comment by melloware** [comment]
OK weird the page looks fine hitting it directly: https://primefaces.github.io/primefaces/jsdocs/index.html

but when accessed through the Docsify Vue application the URL is not resolving correctly. to the above for any of the asset links like `<link rel="stylesheet" href="assets/css/main.css">` its rendering /primefaces/assets instead of /primefaces/jsdocs/assets.  So this may be something weird with Docsify.

**Comment by blutorange** [comment]
Alright, now I know what's going on.

* Docisfy processes internal links and will modify external links
* That why I had added this to make the links to the jsdocs work: https://github.com/primefaces/primefaces/blob/master/docs/9_0/index.html#L35-L38
* But I only added it for the PF 9.0 docs directory, since the jsdocs are for PF 9+
* The link in the opening post is from the PF 8 docs

It seems a jsdocs links was added to the PF 8 docs page for DataTable in a recent commit. Not sure if we want that link for the PF 8 docs. @melloware
